web/src/client/js/components/DocumentUsers/DocumentUsersContainer.js

```diff
@@ -1,23 +1,29 @@
-import React, { useEffect, useState } from 'react'
+import React, { useEffect } from 'react'
 import PropTypes from 'prop-types'
 import { connect } from 'react-redux'
 import { useParams } from 'react-router-dom'
 
 import { currentUserId } from 'Libs/currentIds'
 import { fetchUsersWithIds } from 'Actions/user'
-import useDocumentSocket from 'Hooks/useDocumentSocket'
-import useSyncDocumentRoom from 'Hooks/useSyncDocumentRoom'
+import { emitJoinDocumentRoom, emitLeaveDocumentRoom } from 'Actions/userSync'
 
 import DocumentUsersComponent from './DocumentUsersComponent'
 
-const DocumentUsersContainer = ({ fetchUsersWithIds, usersById, usersLoadedById }) => {
+const DocumentUsersContainer = ({ fetchUsersWithIds, usersById, usersLoadedById, activeDocumentUsers, emitJoinDocumentRoom, emitLeaveDocumentRoom }) => {
   const { documentId } = useParams()
 
-  useSyncDocumentRoom(documentId)
-
-  const { state: socketState } = useDocumentSocket()
+  // User Sync for document room join/leave
+  useEffect(() => {
+    if (document) {
```

Comment:
```suggestion
    if (documentId) {
```

I don't see where document is defined here, is this supposed to be documentId?